src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -101,12 +95,13 @@ def get(self, request, organization):
 
 class OrganizationEventsRelatedIssuesEndpoint(OrganizationEventsEndpointBase, EnvironmentMixin):
     def get(self, request, organization):
-        with sentry_sdk.start_span(op="discover.endpoint", description="filter_params") as span:
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        with sentry_sdk.start_span(op="discover.endpoint", description="find_lookup_keys") as span:
             span.set_data("organization", organization)
```

Comment:
This uses `set_data` rather than `set_tag` to attach the organization. Should this be unified?

Reply:
Mark's comment [here](https://github.com/getsentry/sentry/pull/20849/files#r490976642) raises a good point, I'm just going to remove this instead since we already have the org at event level